.pre-commit-config.yaml

```diff
@@ -43,33 +43,29 @@ repos:
     hooks:
       - id: pylint
         args: [
-          "-rn", # Only display messages
-          "--rcfile=tests/pylintrc", # Link to your config file
-        ]
+            "-rn", # Only display messages
+            "--rcfile=tests/pylintrc", # Link to your config file
+          ]
 
   - repo: https://github.com/psf/black
     rev: 23.3.0
     hooks:
       - id: black
-        args: [
-          --line-length=120,
-          --check,
-          --diff,
-        ]
+        args: [--line-length=120, --check, --diff]
```

Comment:
please revert the auto IDE formatting changes (unless you think they are needed)